src/applications/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -12,7 +12,7 @@ export const ROWS_PER_PAGE = 10;
 export const APPEAL_ACTIONS = {
   original: 'original',
   postRemand: 'post_remand',
-  postCavcRemand: 'post_cavc_remand',
+  postCavcRemand: 'cavc_remand',
```

Comment:
Ahh, I was wondering if it was something like this. The details are fuzzy now, but I think we got this from the swagger doc we were working from at the time. Any idea when that changed?

Also, do the rest of these look good?

Reply:
You're right. Caseflow is wrong. vets-api is expecting the same thing. I'm going to close this and fix on the Caseflow end.